lib/datadog/core/telemetry/event.rb

```diff
@@ -242,15 +245,31 @@ def initialize(changes, origin)
           end
 
           def payload(seq_id)
-            {
-              configuration: @changes.map do |name, value|
-                {
-                  name: name,
-                  value: value,
-                  origin: @origin,
-                }
-              end
-            }
+            @seq_id = seq_id
```

Comment:
The instance variable write here I think is unexpected, if this was intentional a comment would be helpful to clarify the reason for it.